### PR TITLE
LPS-197871 Inconsistency in the Treeview sidebar

### PR DESCRIPTION
**Pull request by danvb112**
Related Issue: [LPS-197871](https://liferay.atlassian.net/browse/LPS-197871)

### CODE REVIEW

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss [hunk 1]
```diff
@@ -42,6 +46,15 @@
...
+		.lfr-objects__model-builder-left-sidebar-show-folders-button {
+			display: flex;
+			visibility: hidden;
+
+			&-disabled {
+				visibility: visible;
+			}
+		}
+
 		&:hover {
```
Comment (rebsilva) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss:58]: bellow the `&:hover`, change this code:

```
&-item {
		border: 3px solid #80acff;
		border-radius: 8px;
	}
```
to this:

```
&-item {
		border: 2px solid #80acff;
		border-radius: 8px;
	}
```

Comment (rebsilva) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ModelBuilder/LeftSidebar/LeftSidebar.scss:58]: the ticket asks for the border to be `2px`

**Review by rebsilva — CHANGES_REQUESTED** [review]
hey @danvb112! just asked for a small change, if you have questions please let me know!

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:test:relevant

**Review by rebsilva — APPROVED** [review]
LGTM! :tada: great job @danvb112 :cherry_blossom:

**Comment by danvb112** [comment]
ci:forward

**Comment by danvb112** [comment]
ci:forward:force

**Pull request closed without merging**